DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: clusterPower
-Version: 0.7.1
+Version: 0.7.2
```

Comment:
**Question:** Is there a "standard" way to indicate this version is a Delfi version and not a `clusterPower` version ?

Reply:
This is not a show stopper, so I am approving.